src/components/Image/Image.scss

```diff
@@ -0,0 +1,7 @@
+.hidden {
```

Comment:
I added the CSS because of the re-rendering the image would show a broken image icon in the browser.